account_fiscal_year_closing_reporting/models/general_ledger_report_compute.py

```diff
@@ -0,0 +1,375 @@
+# Copyright 2020 Sergio Corato <https://github.com/sergiocorato>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+
+
+class GeneralLedgerReportCompute(models.TransientModel):
+    _inherit = 'report_general_ledger'
+
+    def _get_account_sub_subquery_sum_amounts(
+            self, include_initial_balance, date_included):
+        sub_subquery_sum_amounts = super()._get_account_sub_subquery_sum_amounts(
+            include_initial_balance=include_initial_balance,
+            date_included=date_included)
+        for ml_date in ['AND ml.date < %s', 'AND ml.date <= %s']:
+            sub_subquery_sum_amounts = sub_subquery_sum_amounts.replace(
+                ml_date,
+                """
+            INNER JOIN account_move m ON ml.move_id = m.id
+               %s
+               AND m.closing_type not in ('closing', 'opening', 'loss_profit')
+                """
+                % ml_date
+            )
+        return sub_subquery_sum_amounts
+
+    def _get_partner_sub_subquery_sum_amounts(
+            self, only_empty_partner, include_initial_balance, date_included):
+        sub_subquery_sum_amounts = super()._get_partner_sub_subquery_sum_amounts(
+            only_empty_partner=only_empty_partner,
+            include_initial_balance=include_initial_balance,
+            date_included=date_included)
+        for ml_date in ['AND ml.date < %s', 'AND ml.date <= %s']:
+            sub_subquery_sum_amounts = sub_subquery_sum_amounts.replace(
+                ml_date,
+                """
+            INNER JOIN account_move m ON ml.move_id = m.id
+               %s
+               AND m.closing_type not in ('closing', 'opening', 'loss_profit')
+                """
+                % ml_date
+            )
+        return sub_subquery_sum_amounts
+
+    # this function is changed only adding lines:
+    # AND
+    # m.closing_type not in ('closing', 'opening', 'loss_profit')
+    def _inject_line_not_centralized_values(
```

Comment:
@JordiBForgeFlow @pedrobaeza @eLBati I didn't found a way to inherit this function to do [this](https://github.com/OCA/account-closing/pull/134/files#diff-882b756955caf29ae168b87fdc00d98aR308-R309) simple change to limit the account moves to be considered, so I do this ugly copy... If there is a better way, it'll be welcome!

Reply:
So, at the moment, I can't see other ways to do this

Reply:
@sergiocorato could you just add a comment to the modified lines ?